src/main/java/com/google/sps/servlets/LocationsServlet.java

```diff
@@ -47,23 +49,54 @@
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
 
     String office = request.getParameter("office");
+    long startDate = (long) Double.parseDouble(request.getParameter("start"));
```

Comment:
why is the data type long? can we use java.time library? https://docs.oracle.com/javase/8/docs/api/java/time/Instant.html